pkg/services/cloudprovider/csi.go

```diff
@@ -660,6 +648,22 @@ func ConfigForCSI(vsphereCluster v1alpha3.VSphereCluster, cluster clusterv1.Clus
 	return config
 }
 
+func CSIFeatureStatesConfigMap() *corev1.ConfigMap {
+	return &corev1.ConfigMap{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: corev1.SchemeGroupVersion.String(),
+			Kind:       "ConfigMap",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      CSIFeatureStateConfigMapName,
+			Namespace: CSINamespace,
+		},
+		Data: map[string]string{
+			"csi-migration": "false",
```

Comment:
@gab-satchi - where this configmap is being consumed

Reply:
This ConfigMap is the new way to define the csi feature state switches. It was a part of this PR and [here's](https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/487/files#diff-a09d3fa0ab9f566575744f1cae0f65b511bc2093e2029c4204452eb87f4e6ba1R51-R65) where it gets consumed.